lib/Fmt_ast.ml

```diff
@@ -2219,6 +2219,12 @@ and fmt_expression c ?(box = true) ?(pro = noop) ?eol ?parens
             ; _ }
           when not (Std_longident.is_infix id) ->
             has_attr && parens
+        | Exp {pexp_desc= Pexp_tuple tuples; _}
```

Comment:
Instead of adding more special cases into `Fmt_ast`, we handle parentheses in `Ast.ml`: https://github.com/ocaml-ppx/ocamlformat/pull/2733

Reply:
👍 no problem. Your pull request makes a lot of sense 😄